### PR TITLE
Changes to draggable objects and CT footer

### PR DESCRIPTION
**Pull request by kirbyimk**
Made changes to files, substituting onDrag for onDragStart to fix issues with draggable objects. These include the combat tracker turn flag and the Friendly/Neutral/Foe markers. I think I've possibly duplicated the change that Dan has done regarding dragging the character from the character list to the combat tracker.
Also changes the locations of the combat tracker footer objects to fir them within the CT graphic.

### CODE REVIEW

File: templates_common.xml [hunk 1]
```diff
@@ -362,7 +362,7 @@
 					setHoverCursor("hand");
 				end
 			
-				function onDrag(button, x, y, draginfo)
+				function onDragStart(button, x, y, draginfo)
 					if dragging then
 						return true;
 					end
```
Comment (phantomwhale) [inline templates_common.xml:368]: Anything relating to the "dragging" variable can do now, e.g. the three lines above, and wherever is it set to true

**Comment by phantomwhale** [comment]
The changes look good - but could we also remove any uses of the "dragging" variable (as this was only required when onDrag( ) got called repeatedly) ?

**Comment by kirbyimk** [comment]
I'll close this then and resubmit when removed the draggable references

**Comment by phantomwhale** [comment]
Think you can just make the changes and "modify" the pull request - no need for a new one.

**Pull request closed without merging**